sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/KeyOperationParameters.java

```diff
@@ -24,6 +24,24 @@
     @JsonProperty(value = "value", required = true)
     private Base64Url value;
 
+    /**
+     * Initialization vector for symmetric algorithms.
+     */
+    @JsonProperty(value = "iv")
+    private byte[] initializationVector;
+
+    /**
+     * Additional data to authenticate but not encrypt/decrypt when using authenticated crypto algorithms.
+     */
+    @JsonProperty(value = "aad")
+    private byte[] additionalAuthenticatedData;
+
+    /**
+     * The tag to authenticate when performing decryption with an authenticated algorithm.
+     */
+    @JsonProperty(value = "tag")
+    private byte[] tag;
```

Comment:
Use "authenticationTag" (yes, "authentication"; keep "authenticated" for `additionalAuthenticatedData`) for all public APIs here as well. "tag" would just be confusing with all the other instances of "tag" or "tags" used throughout the SDKs.